src/TrueLayer/Payments/Model/GetPaymentResponse.cs

```diff
@@ -76,19 +76,21 @@ public record Authorizing : PaymentDetails;
 
         /// <summary>
         /// Represents a payment that has been authorized by the end user
+        /// <param name="CreditableAt">The date and time that TrueLayer determined that the payment was ready to be credited</param>
         /// </summary>
         /// <returns></returns>
         [JsonDiscriminator("authorized")]
-        public record Authorized : PaymentDetails;
+        public record Authorized(DateTime? CreditableAt) : PaymentDetails;
```

Comment:
Is this correct? It does not seems so based on our public docs? I can't see the field on the `authorized` payment details on the API ref

Reply:
yeah, it's quite new, probably the docs are not updated, I'm checking with Risk

Reply:
It's confirmed, we need to update docs